integration-tests/src/test/java/io/quarkiverse/quinoa/it/QuinoaCodeStartTest.java

```diff
@@ -8,26 +8,26 @@
 
 import io.quarkus.devtools.testing.codestarts.QuarkusCodestartTest;
 
-public class QuinoaCodestartTest {
+public class QuinoaCodeStartTest {
```

Comment:
??? :)

Reply:
Sorry the case seemed weird to me that its CodeStart but the s was lowercase it looked weird to my eyes :)

Reply:
Ok we use Codestart as a single word (a codestart) everywhere, so better keep it that way..

Reply:
OK let me revert that change then.  I didn't realize that it just look funny to me!